### PR TITLE
pull register from latest was Hongyok 

### PR DESCRIPTION
**Pull request by satanakorn**

### CODE REVIEW

**Comment by satanakorn** [comment]
Hongyok Add profile connect with backend and modify hackathon and connect with backend

File: frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React</title>
```
Comment (Celesca) [inline frontend/index.html:7]: Change title to `Hack2gather`

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import './App.css';
+import { useEffect } from 'react';
+import homebg from './assets/homebg.jpg';
+import { TypeAnimation } from 'react-type-animation';
+import { Link } from 'react-router-dom';
+
+function App() {
+  useEffect(() => {
+    console.log('Hello World');
+  });
```
Comment (Celesca) [inline frontend/src/App.jsx:10]: Remove unused useEffect

File: frontend/src/Register.jsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
+// import { FaUser, FaLock } from "react-icons/fa";
```
Comment (Celesca) [inline frontend/src/Register.jsx:1]: Remove unused imports

File: frontend/src/hackathon.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import { Link } from 'react-router-dom';
+import { useEffect } from 'react';
+import  { useState } from 'react';
+
+const Hackathon = () => {
+  const [Hackathonlist, setHackathonlist] = useState([]);
+  
+  const getHackathon = async () => {
+    try {
+      const response = await Axios.get('http://localhost:3307/hackathon');
```
Comment (Celesca) [inline frontend/src/hackathon.jsx:11]: use `localhost:3000`

Comment (Celesca) [reply frontend/src/hackathon.jsx:11]: Nice to have : Try using the .env in frontend folder to store the apiUrl path to easier change in future.

For example : 
`const apiUrl = import.meta.env.BACKEND_URL;`

**Review by Celesca — CHANGES_REQUESTED** [review]
Fix these changes and commit again.

**Review by Celesca — APPROVED** [review]
LGTM

**Pull request merged**